tests/src/ExtensionTest.php

```diff
@@ -6,7 +6,7 @@
 use PHPUnit_Framework_TestCase;
 use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
 use Symfony\Component\Config\Definition\Builder\NodeBuilder;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\Definition\ScalarNode;
```

Comment:
@forceedge01 The 'testConfigure' test in this script is breaking, not my doing. I noticed that there was a missing use statement for ScalarNode which I've put in but this hasn't fixed the issue.

Error message as follows... will leave this for you :)
Trying to configure method "scalarNode" which cannot be configured because it does not exist, has not been specified, is final, or is static
 /Users/chris/Sites/genesis/tests/src/ExtensionTest.php:46